components/centraldashboard/frontend/index/index.html

```diff
@@ -25,6 +25,7 @@ <h1>Kubeflow</h1>
         <a href="/hub/" class="mdl-layout__tab is-active mdl-color-text--white">JupyterHub</a>
         <a href="/tfjobs/ui/" class="mdl-layout__tab mdl-color-text--white">TFJob Dashboard</a>
         <a href="/k8s/ui/" class="mdl-layout__tab mdl-color-text--white">k8s Dashboard</a>
+        <a href="/katib/" class="mdl-layout__tab mdl-color-text--white">katib</a>
```

Comment:
Did you test this? I think we want to redirect to katib/projects/

Reply:
/katib/ automatically redirects to /katib/projects so I left it like that.